src/components/elements/screen-centered.tsx

```diff
@@ -0,0 +1,19 @@
+"use client"
+
+import {HTMLAttributes, useCallback, useEffect, useRef, useState} from "react"
+import {useEventListener} from "usehooks-ts"
+
+const ScreenCentered = ({children}: HTMLAttributes<HTMLDivElement>) => {
+  const ref = useRef<HTMLDivElement>(null)
+  const [translate, setTranslate] = useState(0)
+  const calculateTranslation = useCallback(() => setTranslate(ref.current?.getBoundingClientRect().x || 0), [])
+  useEventListener("resize", calculateTranslation)
+  useEffect(() => calculateTranslation(), [calculateTranslation])
+
+  return (
+    <div ref={ref} className="relative left-1/2 w-screen -translate-x-1/2">
```

Comment:
```suggestion
    <div ref={ref} className="relative left-1/2 w-screen -translate-x-1/2 m-0">
```
Just this little change to remove the white space between banners and we should be good to go!